.github/workflows/test_all_in_one_common.yml

```diff
@@ -0,0 +1,90 @@
+      - name: Install-Podman
+        run: sh -c "sudo apt-get update && sudo apt-get -y install podman"
+      - uses: actions/checkout@v3
+      - name: Cache-jar-files
+        uses: actions/cache@v3
+        with:
+          path: java/buildconf/ivy/repository/
+          key: ${{ runner.os }}-build-cache-uyuni-jars-${{ hashFiles('**/java/buildconf/ivy/*.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-build-cache-uyuni-jars-
+
+      - name: Cache-obs-to-maven files
+        uses: actions/cache@v3
+        with:
+          path: java/.obs-to-maven-cache
+          key: ${{ runner.os }}-build-cache-uyuni-obs-to-maven-${{ hashFiles('**/java/buildconf/ivy/obs-maven-config.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-build-cache-uyuni-obs-to-maven-
+      - name: Cache-nodejs
+        uses: actions/cache@v3
+        with:
+          path: web/html/src/node_modules
+          key: ${{ runner.os }}-build-cache-uyuni-nodejs-${{ hashFiles('**/web/html/src/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-build-cache-uyuni-nodejs-
+      - name: create_tmp
+        run: ./testsuite/podman_runner/01_setup_tmp_dirs.sh
+      - name: create-podman-network
+        run: ./testsuite/podman_runner/02_setup_network.sh
+      - name: start_controller
+        run: ./testsuite/podman_runner/03_run_controller.sh 
+      - name: create_ssh_conf
+        run: ./testsuite/podman_runner/04_setup_ssh_controller.sh 
+      - name: install_gems_in_controller
+        run: ./testsuite/podman_runner/05_install_gems_in_controller.sh
+      - name: start-server-container
+        run: ./testsuite/podman_runner/06_start_server.sh 
+      - name: mgr-setup
+        run: ./testsuite/podman_runner/07_manager_setup.sh 
+      - name: build_code
+        run: ./testsuite/podman_runner/08_build_server_code.sh 
+#     - name: copy_ca
+#     run: podman exec uyuni-server-all-in-one-test bash -c "cp /etc/pki/tls/certs/spacewalk.crt /tmp"
+#     - name: update_ca_in_controller
+#       run: podman exec controller-test bash -c "cat /tmp/spacewalk.crt >> /etc/ssl/ca-bundle.pem"
+      - name: sle-sshminion
+        run: ./testsuite/podman_runner/09_run_sshminion.sh 
+      - name: test_from_host
+        run: curl --insecure https://localhost:8443/rhn/help/Copyright.do
+      - name: test_from_container
+        run: podman exec opensusessh curl --insecure https://uyuni-server-all-in-one-test:443/rhn/help/Copyright.do
+      - name: setup_sshd
+        run: ./testsuite/podman_runner/10_setup_sshd.sh 
+      - name: run_cucumber_core
+        run: ./testsuite/podman_runner/11_run_core_tests.sh 
+      - name: sle-salt-minion
+        run: ./testsuite/podman_runner/12_run_salt_sle_minion.sh 
+      - name: rhlike-minion
+        run: ./testsuite/podman_runner/13_run_salt_rhlike_minion.sh 
+      - name: deblike-minion
+        run: ./testsuite/podman_runner/14_run_salt_deblike_minion.sh 
+      - name: accept_keys
```

Comment:
I'm not sure if I understood the meaning of these scripts.
As we do the bootstrap in init_clients, these scripts it's more about "deploying" the instances?
It might need some better naming I guess.

Reply:
init_clients do not bootstrap but just checks that the minions are there. These scripts use salt cli to bootstrap.